.github/ISSUE_TEMPLATE/release.md

```diff
@@ -18,31 +18,29 @@ Please assign the release manager to the issue.
-- [ ] Open a [docs release issue].
-- [ ] Run the [Tag workflow][tag-workflow] on the relevant release branch with the proper release version.
-- [ ] Run the [CI workflow][ci-workflow] on the release branch and verified that the tagged build version exists on the [releases.crossplane.io] `build` channel..
+  - [ ] Created and merged an empty commit to the `master` branch, if required to have it at least one commit ahead of the release branch.
+  - [ ] Run the [Tag workflow][tag-workflow] on the `master` branch with the release candidate tag for the next release `vX.Y+1.0-rc.0`.
+- [ ] Opened a [docs release issue].
+- [ ] Run the [Tag workflow][tag-workflow] on the `release-X.Y` branch with the proper release version. Message suggested, but not required: `Release vX.Y.0`
+- [ ] Run the [CI workflow][ci-workflow] on the release branch and verified that the tagged build version exists on the [releases.crossplane.io] `build` channel, e.g. `build/release-X.Y/vX.Y.0/...` should contain all the relevant binaries.
 - [ ] Run the [Configurations workflow][configurations-workflow] on the release branch and verified  that version exists on [xpkg.upbound.io] for all getting started packages.
   - [ ] `xp/getting-started-with-aws`
-  - [ ] `xp/getting-started-with-with-with-vpc`
+  - [ ] `xp/getting-started-with-aws-with-vpc`
   - [ ] `xp/getting-started-with-azure`
   - [ ] `xp/getting-started-with-gcp`
-- [ ] Run the [Promote workflow][promote-workflow] with channel `stable` on the release branch and verified that the tagged build version exists on the [releases.crossplane.io] `stable` channel.
-- [ ] Published a [new release] for the tagged version, with the same name as the version and descriptive release notes.
-- [ ] Update the [`crossplane/test` repo test workflows][crossplane-test-workflows] to ensure the checkout release branch and helm install version(s) point at the new release
-- [ ] Update the `baseBranches` list in renovate's configuration file, `.github/renovate.json5`, with the new release branch, removing the old unsupported release one.
+- [ ] Run the [Promote workflow][promote-workflow] with channel `stable` on the `release-X.Y` branch and verified that the tagged build version exists on the [releases.crossplane.io] `stable` channel at `stable/vX.Y.0/...`.
+- [ ] Published a [new release] for the tagged version, with the same name as the version and descriptive release notes, taking care of generating the changes list selecting as "Previous tag" `vX.Y-1.0`, so the first of the releases for the previous minor.
+- [ ] Updated the [`crossplane/test` repo test workflows][crossplane-test-workflows] to ensure the checkout release branch and helm install version(s) point at the new release
+- [ ] Checked that the [docs release issue] created previously has been completed.
+- [ ] Updated the [releases table] in the `README.md` and the `baseBranches` list in `.github/renovate.json5` on `master`.
```

Comment:
maybe add back in the extra note we used to have here? e.g. ", removing the now old and unsupported version"

Reply:
oh and explicitly mention "in a single PR" so folks make sure to streamline here :)

Reply:
sounds reasonable, done!